app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -128,12 +128,44 @@ def process_collection_item(item, key)
       new_result
     end
 
+    def find_host_by_provider_id(provider_id)
+      providers = {
+        "gce" => [ManageIQ::Providers::Google::CloudManager, :name],
+        "aws" => [ManageIQ::Providers::Amazon::CloudManager, :uid_ems],
```

Comment:
- symbols as hash keys
- personally I would inline this hash but it is a matter of taste
```
prov, name_field = {
  :gce => [ManageIQ::Providers::Google::CloudManager, :name],
  :aws => [ManageIQ::Providers::Amazon::CloudManager, :uid_ems],
}[scheme.to_sym]
```